easybuild/easyconfigs/r/R/R-3.0.2-goolf-1.4.10-bare.eb

```diff
@@ -0,0 +1,37 @@
+name = 'R'
+version = "3.0.2"
+versionsuffix = '-bare'  # bare, as in no extensions included
+
```

Comment:
minor changes needed here, `easyblock = 'ConfigureMake'` , put this in REVIEW plz!
(not sure if getting in this historic R ec is useful any more)